htpasswd.go

```diff
@@ -118,7 +118,12 @@ func (h *HtpasswdFile) Reload() error {
 		return err
 	}
 	users := make(map[string]string)
+	valid, _ := regexp.Compile(`^[\p{L}.-]+$`)
 	for _, record := range records {
+		if !valid.MatchString(record[0]) {
+			log.Printf("Illegal htpasswd entry for %s.", record[0])
```

Comment:
How about we tell users what we do?
```suggestion
			log.Printf("Ignoring invalid username %q in htpasswd, consists of characters other than letters", record[0])
```